config/settings/local.py

```diff
@@ -1,5 +1,6 @@
 from .base import *  # noqa
 from .base import env
+from .umkc import *
```

Comment:
Find out how to change the settings file used by the .yml. We should use local-umkc.py and production-umkc.py.